atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -959,6 +961,17 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
             }
         }
 
+        if (AtlasDbConstants.hiddenTables.contains(tableRef)) {
```

Comment:
Take a look at the original code -- the list of tables excluded from postfiltering should be different from hiddenTables